src/plugins/cordova-plugin-geolocation/sim-host.js

```diff
@@ -77,6 +88,49 @@ module.exports = function(messages) {
         }
     }
 
+    function mapEventTelemetryHandler() {
+        pendingMouseEvents++;
+        setTimeout(function () {
+            --pendingMouseEvents;
+
+            if (pendingMouseEvents === 0) {
+                telemetry.sendUITelemetry(Object.assign({}, baseProps, { control: 'geo-map-container' }));
+            }
+        }, mouseEventHoldDelay);
+    }
+
+    function registerTelemetryEvents() {
+        document.getElementById('geo-latitude').onchange = telemetry.sendUITelemetry.bind(this, Object.assign({}, baseProps, { control: 'geo-latitude' }));
```

Comment:
Can we create a separate function that takes the element id as a parameter and attaches the `onchange` handler? We could then reuse it for all the element ids.